tests/integration/components/osf-link/component-test.ts

```diff
@@ -0,0 +1,143 @@
+import { render} from '@ember/test-helpers';
+import Ember from 'ember';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+
+module('Integration | Component | osf-link', hooks => {
+    setupRenderingTest(hooks);
+
+    let orgOnError: any;
+    hooks.beforeEach(function() {
+        orgOnError = Ember.onerror;
+    });
+    hooks.afterEach(function() {
+        Ember.onerror = orgOnError;
+    });
+
+    test('it renders the osf-link correctly for an @href', async assert => {
+        await render(hbs`
+            <OsfLink
+                data-test-get-started-button
+                data-analytics-name='Get started button'
+                class='btn btn-primary'
+                local-class='startButton'
+                @target='_blank'
+                @href='https://osf.io'
+            >
+                {{t 'dashboard.getting_started'}}
+            </OsfLink>
+        `);
+        assert.dom('[data-test-get-started-button]').exists();
+        assert.dom('[data-test-get-started-button]').hasText('Getting Started');
+        assert.dom('[data-test-get-started-button]').hasClass('btn');
+        assert.dom('[data-test-get-started-button]').hasClass('btn-primary');
+        assert.dom('[data-test-get-started-button][href$="https://osf.io"]')
+            .exists('The href https://osf.io is not in the dom');
```

Comment:
For qunit tests, we generally put the success case in these messages, because they are visible if you show successful tests as well as if you show errors. For example: 

<img width="622" alt="Screen Shot 2022-10-03 at 11 50 34 AM" src="https://user-images.githubusercontent.com/6599111/193621510-cae0420d-653b-4de9-b59d-3e8ddc39c884.png">

So I would recommend for this and the next one to say, 'is in the dom' instead of 'is not in the dom'

Reply:
So this is what it looks like in server mode when you show passed tests:

<img width="702" alt="Screen Shot 2022-10-03 at 4 33 39 PM" src="https://user-images.githubusercontent.com/6599111/193677101-fa2856b5-619c-4e6a-a0bc-98ea70d754e2.png">

I agree that it doesn't show on the command line, or if you're not showing passed tests on the server, but this is why we write the passing case rather than the failing case.

Reply:
It's not me who does the output; it's the qunit way. The output is for the description of the attempted assert more than the success or failure of the test. For example, with the built-in messages, it will still say what it was trying to do, but failed to:

<img width="1188" alt="Screen Shot 2022-10-03 at 4 50 50 PM" src="https://user-images.githubusercontent.com/6599111/193679953-99efbd0c-254b-4408-8128-4db55f5f2de2.png">